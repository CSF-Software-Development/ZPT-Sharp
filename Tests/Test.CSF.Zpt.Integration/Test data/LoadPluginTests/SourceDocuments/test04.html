<html>
<head>
<title>Load plugin test 4</title>
</head>
<body>
<div>
  <ul>
    <li tal:repeat="item here/Items">
      <div tal:define="docName item/DocumentName;
                       doc load:here/Documents/?docName/macros/content"
           tal:repace="doc">
        This content should be replaced by the rendered document
      </div>
    </li>
  </ul>
</div>
</body>
</html>