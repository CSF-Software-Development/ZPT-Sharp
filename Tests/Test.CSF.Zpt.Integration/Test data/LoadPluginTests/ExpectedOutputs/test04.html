<html>
<head>
<title>Load plugin test 4</title>
</head>
<body>
<div>
  <ul>
    <li>
<p>
  Hello Joseph Bloggs!
  <br />Your item name is 'One'.
</p>
    </li>
    <li>
  <div>
    <p>Content from the macro, before the slot-filler</p>
    <p>
    Here's some slot-filling content
    <br />Your item name is 'Two'.
  </p>
    <p>Content from the macro, after the slot-filler</p>
  </div>
    </li>
    <li>
  <div>
    <p>This is content from a different macro</p>
    <p>
    Here's some slot-filling content
    <br />Your item name is 'Three'.
  </p>
    <p>This is content from a different macro</p>
  </div>
    </li>
  </ul>
</div>
</body>
</html>